02 Charging For Your App/app.rb

```diff
@@ -0,0 +1,207 @@
+require 'shopify_api'
+require 'sinatra'
+require 'httparty'
+require 'dotenv'
+
+class GiftBasket < Sinatra::Base
+  attr_reader :key,:secret,:tokens
+
+  def initialize
+    Dotenv.load
+    @key = ENV['API_KEY']
```

Comment:
key, secret, app_url should probably be defined as constants outside of `initialize`